docs/internal/Element-Type-Specs.md

```diff
@@ -195,35 +195,35 @@ _For displaying blocks of text in the application_
 
 #### `style: none` (default)
 
-![style:none](/images/Element-Type-Specs-textInfo-none.png)
+![style:none](images/Element-Type-Specs-textInfo-none.png)
 
 #### `style: basic`
 
-![style:basic](/images/Element-Type-Specs-textInfo-basic.png)
+![style:basic](images/Element-Type-Specs-textInfo-basic.png)
 
 #### `style: info`
 
-![style:info](/images/Element-Type-Specs-textInfo-info.png)
+![style:info](images/Element-Type-Specs-textInfo-info.png)
 
 #### `style: warning`
 
-![style:warning](/images/Element-Type-Specs-textInfo-warning.png)
+![style:warning](images/Element-Type-Specs-textInfo-warning.png)
```

Comment:
This URL syntax was preventing the images showing up on the wiki. Hopefully this fixes it.